infra/helper.py

```diff
@@ -353,6 +355,33 @@ def get_parser():  # pylint: disable=too-many-statements
   _add_external_project_args(coverage_parser)
   _add_architecture_args(coverage_parser)
 
+  introspector_parser = subparsers.add_parser(
+      'introspector',
+      help='Run a complete end-to-end run of '
+      'fuzz introspector. This involves (1) '
+      'building the fuzzers with ASAN; (2) '
+      'running all fuzzers; (3) building '
+      'fuzzers with coverge; (4) extracting '
+      'coverage; (5) building fuzzers using '
+      'introspector')
+  introspector_parser.add_argument('project', help='name of the project')
+  introspector_parser.add_argument('--seconds',
+                                   help='number of seconds to run fuzzers',
+                                   default=10)
+  introspector_parser.add_argument('source_path',
```

Comment:
Is there any specific reason for not making it `--source_path`?

Reply:
Yes, the reason is to use the same approach as here: https://github.com/google/oss-fuzz/blob/0eb5e001be631f31ec86d3ef03059ae4ccadc8ec/infra/helper.py#L278-L280

Point being the `source_path` is not specified on the command line but rather considered to be the second positional argument (the positional argument after project name).

Reply:
Thanks!